netmanager/src/views/apis/deviceRegistry.js

```diff
@@ -22,6 +22,8 @@ import {
 import { DEVICE_MAINTENANCE_LOG_URI } from "config/urls/deviceMonitoring";
 import { DEVICE_RECENT_FEEDS } from "config/urls/dataManagement";
 
+axios.defaults.headers.common.Authorization = `JWT ${process.env.Authorization}`;
```

Comment:
Can we use the logged in user's Authorization token instead of using a generic token?

Reply:
The current logged in user token https://github.com/airqo-platform/AirQo-frontend/blob/13a51c66b5146ab22500ae58cebf71a0be338d1d/netmanager/src/redux/Join/actions.js#L409

Reply:
Ideally, line 25 is not needed if a user is already logged in

Reply:
@123MwanjeMike I have removed them. Thanks.